packages/console/src/components/CopyToClipboard/index.tsx

```diff
@@ -30,6 +31,7 @@ function CopyToClipboard({
   hasVisibilityToggle,
   variant = 'contained',
   size = 'default',
+  isWordWrapAllowed = false,
```

Comment:
Is it possible to set the styles from the parent element scope?

Reply:
No, the `CopyToClipboard` component is a complex component and the content element is nested deeply in the hierarchy.